egs2/commonvoice/asr1/README.md

```diff
@@ -1,5 +1,105 @@
+|decode_without_lm_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/test_fr|15621|151227|62.3|34.0|3.7|3.1|40.8|92.8|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/dev_fr|15621|939088|91.8|3.7|4.4|2.6|10.8|85.5|
+|decode_asr_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/test_fr|15621|952803|90.9|4.3|4.8|3.0|12.1|87.9|
+|decode_without_lm_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/dev_fr|15621|939088|90.8|3.9|5.3|2.8|11.9|91.3|
+|decode_without_lm_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/test_fr|15621|952803|89.9|4.5|5.6|3.2|13.3|92.8|
+
+### TER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/dev_fr|15621|720834|89.0|6.9|4.2|2.6|13.6|85.5|
+|decode_asr_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/test_fr|15621|730898|87.7|7.9|4.4|3.0|15.3|87.9|
+|decode_without_lm_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/dev_fr|15621|720834|87.4|7.1|5.5|2.4|15.0|91.3|
+|decode_without_lm_lm_lm_train_lm_fr_bpe350_valid.loss.ave_asr_model_valid.acc.ave/test_fr|15621|730898|86.0|8.3|5.7|2.8|16.8|92.8|
+
+<!-- -->
+
+
+We provide the results for the commonvoice **Welsh** dataset with adapters inserted:
+
+
+- date: `Mon Jun  6 23:35:21 EDT 2022`
+- python version: `3.8.10 (default, Mar 15 2022, 12:22:08)  [GCC 9.4.0]`
+- espnet version: `espnet 202204`
+- pytorch version: `pytorch 1.10.1+cu111`
+- Git hash: `a9020e32d602b7427e383c1a889c6b02f471b07c`
+  - Commit date: `Mon Jun 6 13:14:42 2022 -0400`
+
+## asr_oxford_frontend_raw_cy_bpe150_sp
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/dev_cy|2933|28498|33.0|55.4|11.6|6.3|73.3|99.6|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/test_cy|2937|26046|54.0|37.4|8.6|3.8|49.8|77.7|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/dev_cy|2933|156313|72.0|13.8|14.2|7.8|35.8|99.6|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/test_cy|2937|139576|80.2|9.3|10.4|5.2|25.0|77.7|
+
+### TER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/dev_cy|2933|94018|60.8|28.5|10.7|9.3|48.5|99.6|
+|decode_asr_lm_lm_train_lm_cy_bpe150_valid.loss.ave_asr_model_valid.acc.ave/test_cy|2937|83456|72.6|19.4|8.1|6.5|33.9|77.7|
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 ## How to use fusion of front-ends : 
```

Comment:
```suggestion
# How to use fusion of front-ends : 
```
This part is not related to the adapter. So, change the section level to make it independent.

Reply:
@jwrh